sys/meson.py

```diff
@@ -172,6 +172,11 @@ def win_dist(args):
     with open(r2_bat_fname, 'w') as r2_bat:
         r2_bat.write('@"%~dp0\\radare2" %*\n')
 
+    r2_sh_fname = args.install + r'\bin\r2'
+    log.debug('create "%s"', r2_sh_fname)
+    with open(r2_sh_fname, 'w') as r2_sh:
+        r2_sh.write('#!/bin/sh\n$(dirname "$0")/radare2 "$@"\n')
```

Comment:
why not just use a symlink?

Reply:
on Windows, symlinks require their creators to jump through hoops ([either Administrator or Developer Mode](https://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/)) and i don't think `sys/meson.py` can assume that either is enabled